modules/seo-tools.php

```diff
@@ -14,5 +14,21 @@
  */
 
 include dirname( __FILE__ ) . '/seo-tools/jetpack-seo.php';
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+// Suppress SEO Tools output if any of the following plugins is active.
+$jetpack_seo_conflicting_plugins = array(
+	'wordpress-seo/wp-seo.php',
+	'wordpress-seo-premium/wp-seo-premium.php',
+	'all-in-one-seo-pack/all_in_one_seo_pack.php',
+	'all-in-one-seo-pack-pro/all_in_one_seo_pack.php',
+);
+
+foreach( $jetpack_seo_conflicting_plugins as $seo_pluging ) {
+	if ( is_plugin_active( $seo_pluging ) ) {
```

Comment:
It might be best to use our own `get_active_plugins()` here, to account for Multisite networks.